script1.sh

```diff
@@ -0,0 +1 @@
+cat wages.csv | sed 's/,/ /g' | cut -d ' ' -f 1,2 | sed '1d' | sort -k1,1 -k2,2n > gender_yearsExperience.txt
```

Comment:
Close. The output file should contain the unique gender years experience. You just need to pipe through uniq at the end, before writing the file. 
-0.25 pts